pyartcd/pyartcd/pipelines/okd.py

```
@@ -583,10 +585,41 @@ def _get_payload_tag_name(self, image_distgit_key: str, image_metadata: dict) ->
+                f.flush()
+                image_info = await oc_image_info_for_arch_async(
+                    manifest_list_pullspec, go_arch='arm64', registry_config=f.name
+                )
+            arch_digest = image_info.get('digest')
+            if not arch_digest:
+                self.logger.warning('No arm64 digest found in manifest list %s', manifest_list_pullspec)
+                return None
+            # Construct arch-specific pullspec: keep registry/repo, replace digest
+            base = manifest_list_pullspec.split('@')[0]
+            return f'{base}@{arch_digest}'
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)